Tests/js/views/assets/ez-actionbarview-tests.js

```diff
@@ -116,6 +146,36 @@ YUI.add('ez-actionbarview-tests', function (Y) {
         "Should not instantiate deleteActionView": function () {
             this._isNotActionCreated('deleteContent', {});
         },
+
+        "Should disable sendToTrashActionView if root location": function() {
```

Comment:
Regarding merging the two test I told you yesterday, I meant merging them into one `rootLocationTest`, not into the existing viewTest.
You can call or extend the existing `viewTest` to avoid code duplication, let me try to find an example.